build.gradle

```diff
@@ -64,7 +64,8 @@ javadocJar.enabled = false
 ext {
     slf4jVersion = '1.7.7'
     servletApiVersion = '3.1.0'
-    zipkinVersion = '1.7.0'
+    zipkinCoreVersion = '1.7.0'
+    zipkinReporterVersion = '0.2.0'
```

Comment:
Understood. Not too concerned for the moment. If it becomes problematic I can take the shade/shadow steps you mentioned.